variables.tf

```
@@ -160,3 +160,21 @@ variable "properties" {
   default     = {}
   description = "Contents of the server.properties file. Supported properties are documented in the [MSK Developer Guide](https://docs.aws.amazon.com/msk/latest/developerguide/msk-configuration-properties.html)"
 }
+
+variable "storage_autoscaling_target_value" {
+  type        = number
+  default     = 60
+  description = "Percentage of storage used to trigger autoscaled storage increase"
+}
+
+variable "storage_autoscaling_max_capacity" {
+  type        = number
+  default     = null
+  description = "Maximum size autoscaling policy can scale storage. Defaults to broker_volume_size"
+}
+
+variable "storage_autoscaling_disable_scale_in" {
```

We usually use a positive naming convention or it gets confusing

```suggestion
variable "storage_autoscaling_scale_in_enabled" {
```

---

The var name matches the input so I'll resolve this comment.